src/saml20_clj/coerce.clj

```diff
@@ -15,78 +36,78 @@
 (defonce ^:private -init
   (do
     ;; add BouncyCastle as a security provider.
-    (java.security.Security/addProvider (org.bouncycastle.jce.provider.BouncyCastleProvider.))
+    (Security/addProvider (BouncyCastleProvider.))
     ;; initialize OpenSAML
-    (org.opensaml.core.config.InitializationService/initialize)
+    (InitializationService/initialize)
     ;; verify that OpenSAML has the crypto classes it needs
-    (.init (org.opensaml.xmlsec.config.impl.JavaCryptoValidationInitializer.))
+    (.init (JavaCryptoValidationInitializer.))
     nil))
 
 (defprotocol CoerceToPrivateKey
   (->PrivateKey
-    ^java.security.PrivateKey [this]
-    ^java.security.PrivateKey [this ^String algorithm]
+   [this]
+   [this ^String algorithm]
     "Coerce something such as a base-64-encoded string or byte array to a `PrivateKey`. This isn't used directly by
  OpenSAML -- the key must be passed as part of an OpenSAML `Credential`. See `->Credential`."))
 
 (defprotocol CoerceToX509Certificate
-  (->X509Certificate ^java.security.cert.X509Certificate [this]
+  (^java.security.cert.X509Certificate ->X509Certificate [this]
```

Comment:
I'm pretty sure the `:tag` is supposed to go on the argslist and not the method name